examples/demo.ipynb

```diff
@@ -11,11 +11,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For installing cleanvision, you will have to first clone the repo and then install from source.\n",
-    "Steps for installing from cloned repo\n",
-    "```bash\n",
-    "pip install git+https://github.com/cleanlab/cleanvision.git\n",
-    "```\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/cleanlab/cleanvision/blob/main/examples/demo.ipynb) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -U pip\n",
```

Comment:
is this line necessary? Isn't the package being installed from github?

Reply:
It is necessary, otherwise it installs cleanvision as UNKNOWN package. My guess is, it is because I am using pyproject.toml for packaging which is fully supported by later versions of pip